.travis.yml

```diff
@@ -1,5 +1,10 @@
 language: node_js
 before_install:
+  - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
```

Comment:
I have a question, Do we need to update the requirements in our documentation?

Reply:
@warrenv Good question. I can run this in our local-dev without changing anything, so I didn't think so. It appeared to be a Travis issue you helped me resolve

Reply:
maybe can be a `know issue`, but I don't know.

Reply:
I am running Ubuntu as my os and did not need to run this, and I am pretty sure I never have before. Travis is probably running a stripped out version of Ubuntu to reduce server load and they have you add back things.